<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GEO" name="Test from Chome" tests="3" failures="1" timestamp="2 Jul 2021 11:18:41 GMT" time="69.144" errors="0">
  <testcase name="verifyLoginPageTitleTest" time="0.106" classname="com.MyTest.LoginTest">
    <failure type="junit.framework.ComparisonFailure" message="expected:&amp;lt;nopCommerce demo store. Login&amp;gt; but was:&amp;lt;Welcome, Please Sign In!&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<nopCommerce demo store. Login> but was:<Welcome, Please Sign In!>
	at junit.framework.Assert.assertEquals(Assert.java:81)
	at junit.framework.Assert.assertEquals(Assert.java:87)
	at com.MyTest.LoginTest.verifyLoginPageTitleTest(LoginTest.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- verifyLoginPageTitleTest -->
  <testcase name="verifyLoginPageHeader" time="0.255" classname="com.MyTest.LoginTest"/>
  <testcase name="doLoginTest" time="5.244" classname="com.MyTest.LoginTest"/>
</testsuite> <!-- Test from Chome -->
